.changeset/dirty-cooks-explode.md

```diff
@@ -0,0 +1,9 @@
+---
+'@astrojs/mdx': major
+---
+
+Handles the breaking change in Astro where content pages (including `.mdx` pages) no longer respond with `charset=utf-8` in the `Content-Type` header.
+
+For MDX pages without layouts, `@astrojs/mdx` will automatically add the `<meta charset="utf-8">` tag to the page by default. This reduces the boilerplate needed to write with non-ASCII characters. If your MDX pages have a layout, the layout component should include the `<meta charset="utf-8">` tag.
```

Comment:
Is this *only* for the `layout` frontmatter property when it comes to MDX? Or, even if a layout is imported, is this also the case? 

This wording is fine if "without layouts" also means "without importing a component that is being used to wrap content as a layout," but if it's only the frontmatter property, we should distinguish that. (I know the exported properties from a MDX file are different depending on which kind of layout use you have, so maybe something is different here, too?)

Otherwise, I'd be tempted to say a similar, but possibly abbreviated version of what's in the other changeset. The other changeset should be as explicit as possible I think, but the MDX can totally be a tiny bit more summary level (as you've already done!).